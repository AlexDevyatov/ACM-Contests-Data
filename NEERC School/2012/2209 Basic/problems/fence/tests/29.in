8
-55 -32
-25 -60
30 -58
52 -54
54 -49
60 20
26 60
-38 56
-9 55
