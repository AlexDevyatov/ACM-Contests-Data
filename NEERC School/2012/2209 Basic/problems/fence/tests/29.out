9
-55 -32
-25 -60
30 -58
52 -54
54 -49
60 20
26 60
-9 55
-38 56
